src/voice_missed_call_service.js

```diff
@@ -0,0 +1,49 @@
+go.app = function() {
+    var vumigo = require('vumigo_v02');
+    var App = vumigo.App;
+    var EndState = vumigo.states.EndState;
+
+    var GoApp = App.extend(function(self) {
+        App.call(self, 'state_start');
+
+        self.add = function(name, creator) {
+            self.states.add(name, function(name, opts) {
+                return creator(name, opts);
+            });
+        };
+
+        // START STATE
+
+        self.states.add('state_start', function() {
+            var msisdn = go.utils.normalize_msisdn(
+                self.im.user.addr, self.im.config.country_code);
+            return go.utils
+                .get_identity_by_address({'msisdn': msisdn}, self.im)
+                .then(function(identity){
+                    if (identity) {
+                        return go.utils
+                            .resend_all_subscriptions(self.im, identity)
+                            .then(function(count){
```

Comment:
This won't be a `count`, but rather a list of results from each of the web requests to resend the message.